weave/graph.py

```diff
@@ -230,6 +230,8 @@ def op_full_name(op: Op) -> str:
 
 
 def node_expr_str(node: Node) -> str:
+    from . import partial_object
```

Comment:
Is there a reason not to import at the top of the file?

Reply:
Circular import